Client/Frontend/Reader/ReaderModeStyleViewController.swift

```diff
@@ -29,7 +29,20 @@ private struct ReaderModeStyleViewControllerUX {
 // MARK: -
 
 protocol ReaderModeStyleViewControllerDelegate {
-    func readerModeStyleViewController(_ readerModeStyleViewController: ReaderModeStyleViewController, didConfigureStyle style: ReaderModeStyle)
+    func readerModeStyleViewController(_ readerModeStyleViewController: ReaderModeStyleViewController, 
+                                       didConfigureStyle style: ReaderModeStyle)
+    
+    func readerModeStyleViewController(_ readerModeStyleViewController: ReaderModeStyleViewController, 
+                                       didConfigureStyle style: ReaderModeStyle,
+                                       isUsingUserDefinedColor: Bool)
+}
+
+extension ReaderModeStyleViewControllerDelegate {
+    // Can't use default values in protocols, using chained method calls instead
+    func readerModeStyleViewController(_ readerModeStyleViewController: ReaderModeStyleViewController, 
+                                       didConfigureStyle style: ReaderModeStyle) {
+        self.readerModeStyleViewController(readerModeStyleViewController, didConfigureStyle: style, isUsingUserDefinedColor: false)
+    }
```

Comment:
Instead of chaining like you mentioned `Can't use default values in protocols` we should just use your new method. This will look more like: 

```Swift
// Todo: Add a comment explaining that we should keep default value of is as false 
protocol ReaderModeStyleViewControllerDelegate {
    func readerModeStyleViewController(_ readerModeStyleViewController: ReaderModeStyleViewController,
                                           didConfigureStyle style: ReaderModeStyle,
                                           isUsingUserDefinedColor: Bool)
}

```

Reply:
We can then modify at other places like: 

`self.readerModeStyleViewController(ReaderModeStyleViewController(), didConfigureStyle: readerModeStyle, isUsingUserDefinedColor: false)`